C/capnproto/build_tarballs.jl

```diff
@@ -0,0 +1,42 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder
+
+name = "capnp"
+version = "0.7.0"
+
+# Collection of sources required to build FLAC
+sources = [
+    "https://capnproto.org/capnproto-c++-$(version).tar.gz" =>
+    "c9a4c0bd88123064d483ab46ecee777f14d933359e23bff6fb4f4dbd28b4cd41",
+    "./bundled"
+]
+
+version = VersionNumber(version)
```

Comment:
For the record, you can just define
```julia
version = v"0.7.0"
```
above.  It'll be rendered correctly in string interpolation